### PR TITLE
422-after-selecting-a-new-program-on-the-cost-coverage-outcome-page-the-values-in-the-text-boxes-should-reset-to-blank

### PR DESCRIPTION
**Pull request by kitze**
https://trello.com/c/49daMxjr/422-after-selecting-a-new-program-on-the-cost-coverage-outcome-page-the-values-in-the-text-boxes-should-reset-to-blank

### CODE REVIEW

**Comment by Ashalynd** [comment]
In general, these parameters might be saved in the model, or not? @rstuart?

**Comment by Ashalynd** [comment]
@nikgraf can you have a look on this one now?

File: client/source/js/modules/model/cost-coverage-ctrl.js [hunk 1]
```diff
@@ -50,19 +51,13 @@ define(['./module', 'underscore'], function (module, _) {
...
-          });}
-
+      $scope.programs =  _(meta.progs.long).map(function (name, index) {
+        var acronym = meta.progs.short[index];
         return {
           name: name,
-          acronym: meta.progs.short[index],
-          category: categories[index]
+          acronym: acronym,
+          category: 'Other', // it will be read from project_info, once it is synced with meta.programs
```
Comment (nikgraf) [inline client/source/js/modules/model/cost-coverage-ctrl.js]: @Ashalynd is there a card for this?

Comment (Ashalynd) [inline client/source/js/modules/model/cost-coverage-ctrl.js]: @nikgraf I asked @cliffckerr a question about it.

**Pull request merged**